src/com/intel/cancelPreviousBuildsInternal.groovy

```diff
@@ -0,0 +1,35 @@
+// src/com/intel/cancelPreviousBuildsInternal.groovy
+
+package com.intel
+
+/**
```

Comment:
I've just noticed the Doxygen style comments, are we using them throughout?

Reply:
Not yet.